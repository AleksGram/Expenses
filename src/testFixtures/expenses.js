import moment from 'moment';

export default [{
    id: '1',
    description: 'Car',
    note: 'new',
    amount: 5000,
    createdAt: 0
},
{
    id: '2',
    description: 'Wash',
    note: '',
    amount: 50,
    createdAt: moment(0).subtract(4, 'days').valueOf()
},
{
    id: '3',
    description: 'Gym',
    note: 'membership',
    amount: 100,
    createdAt: moment(0).add(4, 'days').valueOf()
}]
