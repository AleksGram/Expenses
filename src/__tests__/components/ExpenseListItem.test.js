import React from 'react';
import { shallow } from 'enzyme';
import  ExpenseListItem  from '../../components/ExpenseListItem';
import expenses from '../../testFixtures/expenses';

test('ExpenseListItemRender', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]}/>)

    expect(wrapper).toMatchSnapshot();
})
