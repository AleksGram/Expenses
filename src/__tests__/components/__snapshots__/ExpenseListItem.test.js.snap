// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseListItemRender 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h1>
      Car
    </h1>
  </Link>
  <p>
    5000
     - 
    0
  </p>
</div>
`;
